rust_snuba/src/processors/replays.rs

```diff
@@ -209,9 +212,15 @@ pub fn deserialize_message(
     Ok((rows, replay_message.start_time))
 }
 
+#[derive(Debug, Deserialize)]
+enum RawReplayPayload {
```

Comment:
This needs a `#[serde(untagged)]` annotation.

Reply:
Done 👍